### ISSUE TITLE
I201 Is Only Enforced When Selected

### ISSUE THREAD

**Issue by pwgraham91** [PyCQA/flake8-import-order#119]
flake8 only throws errors on I201 when `--select I201` is used

example:
my_file.py
```
import logging
import wtforms
import wtforms.validators as validators
```
`flake8 my_file.py`
(returns nothing)

`flake8 --select I201 my_file.py`
`my_file.py:4:1: I201 Missing newline before sections or imports. I201 Missing newline before sections or imports.`

**Comment by pgjones** [comment]
I think there must be something extra in your setup that explains this, as I can't reproduce this. Is there something in your `setup.cfg` for example?

**Comment by pwgraham91** [comment]
I don't have setup.cfg

**Comment by pwgraham91** [comment]
I just tried in my /Documents directory with just flake8 and flake8-import-order and got the same result

**Comment by pgjones** [comment]
Hmm, do any of [these](http://flake8.pycqa.org/en/latest/user/configuration.html) configuration files exist?

**Comment by pwgraham91** [comment]
Unfortunately not

**Comment by pwgraham91** [comment]
```
flake8==3.5.0
flake8-import-order==0.14.3
```

**Comment by pgjones** [comment]
I can reproduce this if I use Python2, and it seems to be insensitive to the flake8-import-order version, but sensitive to the flake8 version in that it works for flake8 <= 3.3.0. I think it is this [issue](https://gitlab.com/pycqa/flake8/issues/318) that explains the change and the `I10` [here](https://github.com/PyCQA/flake8-import-order/blob/master/setup.py#L55) that causes the bug. Will fix.

**Comment by pgjones** [comment]
This should be fixed in master, can you check?

**Comment by pgjones** [comment]
Fix confirmed in #121.

**Comment by pwgraham91** [comment]
verified it works. Thank you!!
